tests/integration_tests/side_od_crop_augmentation.py

```diff
@@ -0,0 +1,71 @@
+import random
+from typing import Optional
+import numpy as np
+
+
+class DetectionRandomSideCrop:
+    """Crop a random part of the input.
+
+    Args:
+        height (int): height of the crop.
+        width (int): width of the crop.
+        p (float): probability of applying the transform. Default: 1.
+
+    Targets:
+        image, mask, bboxes, keypoints
+
+    Image types:
+        uint8, float32
+    """
+
+    def __init__(self, rel_x: float = 0.5, side: Optional[str] = None, p: float = 1.0):
```

Comment:
```suggestion
    def __init__(self, ,max_rel_x: float = 0.5, side: Optional[str] = None, p: float = 1.0):
```
esto sería el máximo relativo, no el relativo. porque se samplea uniformemente entre 0 y esto.